libs/ui-lib/src/lib/custom-file-upload/custom-file-upload.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ChangeEvent, ReactNode } from 'react';
```

Comment:
Please move styling relevent to the custom-file-upload.tsx component to `libs/ui-lib/src/lib/custom-file-upload/custom-file-upload.scss`